.github/actions/setup-dotnet/action.yml

```diff
@@ -0,0 +1,8 @@
+name: Setup .NET
```

Comment:
Did you consider using setup-dotnet directly?

Reply:
Yes, but since we want to be using the same version in all of our actions I found this be a good way to do that.

Reply:
Yes. Makes sense to keep it.